spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -69,6 +71,9 @@
 	@Autowired
 	private ConfigurableApplicationContext context;
 
+	@Autowired
```

Comment:
Tasks provides one by default.  See the `ResourceLoadingAutoConfiguration`.  We should always have one.